pkgs/development/libraries/xed/default.nix

```diff
@@ -0,0 +1,44 @@
+{ stdenv, fetchFromGitHub, python3Packages }:
+
+let
+  # mbuild is a custom build system used only to build xed
+  mbuild = python3Packages.buildPythonPackage rec {
+    pname = "mbuild";
+    version = "0.2496-dev";
+
+    src = fetchFromGitHub {
+      owner = "intelxed";
+      repo = "mbuild";
+      rev = "3e8eb33aada4153c21c4261b35e5f51f6e2019e8";
+      sha256 = "0yamgzkzw4v6x1a857psw9f7i62ydgd0zaqrf33dbdg8hfd2mq3q";
+    };
+  };
+
+in stdenv.mkDerivation rec {
+  pname = "xed";
+  version = "12.0.1";
+
+  src = fetchFromGitHub {
+    owner = "intelxed";
+    repo = "xed";
+    rev = version;
+    sha256 = "07zfff8zf29c2n0wal87hiqfq3cwcjn80zz78mz0nyjfj09nd39f";
+  };
+
+  nativeBuildInputs = [ mbuild ];
+
+  # this will build, test and install
+  buildPhase = ''
+    ./mfile.py test --prefix $out
```

Comment:
```suggestion
  buildPhase = ''
    # this will build, test and install
    ./mfile.py test --prefix $out
```

Reply:
hmm, I remember someone suggesting doing the opposite on one on my PRs, is there a preferred way?

Reply:
The comment belongs to this line of code. If we later add more commands to the buildPhase the comment is wrong.